model/src/main/kotlin/Error.kt

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2017-2018 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.model
+
+import com.fasterxml.jackson.core.JsonParser
+import com.fasterxml.jackson.databind.DeserializationContext
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer
+
+import java.time.Instant
+
+/**
+ * An error that occured while executing ORT.
+ */
+data class Error(
+        /**
+         * The timestamp of the error.
+         */
+        val timestamp: Instant = Instant.now(),
+
+        /**
+         * A description of the error source, e.g. the tool that caused the error.
+         */
+        val source: String,
+
+        /**
+         * The error message.
+         */
+        val message: String
+) {
+    override fun toString() = "$timestamp: $source - $message"
+}
+
+class ErrorDeserializer : StdDeserializer<Error>(Error::class.java) {
+    override fun deserialize(p: JsonParser, ctxt: DeserializationContext): Error {
+        val node = p.codec.readTree<JsonNode>(p)
+        return if (node.isTextual) {
+            // For backward-compatibility if only "allowDynamicVersions" / "allow_dynamic_versions" is specified.
+            Error(Instant.EPOCH, "", node.textValue())
```

Comment:
For the timestamp I would prefer to have "n/a" here instead of the Unix epoch. When I see the latter in some logging output, I assume a configuration problem on the machine that generated the log. Maybe convert the Unix epoch to "n/a" in `Error.toString()`?

Reply:
Good idea with `toString()`, done.